lib/graphql/interface_type.rb

```diff
@@ -23,4 +23,14 @@ def kind
   def possible_types
     @possible_types ||= []
   end
+
+  # @return [GraphQL::Field] The defined field for `field_name`
+  def get_field(field_name)
+    self.fields[field_name]
```

Comment:
Seems like kind of a waste, `all_fields` flattens the hash to an array, then `get_fields` picks a member out of the array by name. We can save a few steps by looking up directly from the hash!